third_party/cxx/blobstore/build.rs

```diff
@@ -1,9 +1,16 @@
 fn main() {
+    let src_dir = {
```

Comment:
```suggestion
    // Handle $CARGO_TARGET_DIR being set.
    let src_dir = {
```

Reply:
Looking at this, you could consider writing it more directly as

```rust
// Handle $CARGO_TARGET_DIR being set.
let mut src_dir = std::env::var_os("CARGO_TARGET_DIR").unwrap_or("../../../target".into());
src_dir.push("/cxxbridge/demo/src");
```

Also, do you know how this relates to the `$OUT_DIR` variable which [Cargo sets](https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates)?

Reply:
Done.

As I understand it, `$OUT_DIR` refers to a temporary directory, and is an output of Cargo; `$CARGO_TARGET_DIR` is an input to Cargo and might change what `$OUT_DIR` ends up being (but I'm not sure, I don't touch build scripts often... unless they break).